ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -265,21 +342,29 @@ Column {
 //            nextMessageIndex: root.messageStore.nextMessageIndex
 //            nextMsgTimestamp: root.messageStore.nextMsgTimestamp
             onClicked: {
-                root.chatsModel.messageView.hideLoadingIndicator();
+                // Not Refactored Yet
+//                root.chatsModel.messageView.hideLoadingIndicator();
             }
             onTimerTriggered: {
-                root.chatsModel.requestMoreMessages(Constants.fetchRangeLast24Hours);
+                // Not Refactored Yet
+//                root.chatsModel.requestMoreMessages(Constants.fetchRangeLast24Hours);
             }
         }
     }
 
     Component {
         id: channelIdentifierComponent
-        ChannelIdentifierView {
-            store: root.rootStore
-            profileImage: profileImageSource
-            authorCurrentMsg: root.authorCurrentMsg
+        Rectangle {
+            color: "blue"
+            width: 100
+            height: 100
         }
```

Comment:
Is this a leftover?

Reply:
@jrainville I left that intentionally since `ChannelIdentifierView` qml component is not refactored yet but `channelIdentifierComponent` is used in `messageLoader` and since `Component` cannot be empty I just commented `ChannelIdentifierView` and add that dummy rectangle in order to have compilable code. Will work on that component today.